pydis_site/apps/content/resources/guides/pydis-guides/contributing/bot.md

```diff
@@ -36,399 +36,142 @@ If your bot fails to start with a `PrivilegedIntentsRequired` exception, this in
-        status_offline: "<:status_offline:�>"
-        status_online:  "<:status_online:�>"
-
-        trashcan: "<:trashcan:�>"
-
-##### <<  Optional - If you don't care about the filtering, help channel and py-news cogs, ignore the rest of this file  >> #####
-filter:
-    # What do we filter?
-    filter_domains:        true
-    filter_everyone_ping:  true
-    filter_invites:        true
-    filter_zalgo:          false
-    watch_regex:           true
-    watch_rich_embeds:     true
-
-    # Notify user on filter?
-    # Notifications are not expected for "watchlist" type filters
-    notify_user_domains:        false
-    notify_user_everyone_ping:  true
-    notify_user_invites:        true
-    notify_user_zalgo:          false
-
-    # Filter configuration
-    offensive_msg_delete_days: 7     # How many days before deleting an offensive message?
-    ping_everyone:             true
-
-    # Censor doesn't apply to these
-    channel_whitelist:
-        - *DEV_ADMINS_MODS
-        - *DEV_BB_LOGS
-        - *DEV_ALL_LOGS
-        - *DEV_LOUNGE_HELPERS_ORG
-
-    role_whitelist:
-        - *ADMINS_ROLE
-        - *HELPERS_ROLE
-        - *MODS_ROLE
-        - *OWNERS_ROLE
-        - *PY_COMMUNITY_ROLE
-        - *PY_PARTNER_ROLE
-
-help_channels:
-    enable: true
-
-    # Minimum interval before allowing a certain user to claim a new help channel
-    claim_minutes: 1
-
-    # Roles which are allowed to use the command which makes channels dormant
-    cmd_whitelist:
-        - *HELPERS_ROLE
-
-    # Allowed duration of inactivity before making a channel dormant
-    idle_minutes: 1
-
-    # Allowed duration of inactivity when channel is empty (due to deleted messages)
-    # before message making a channel dormant
-    deleted_idle_minutes: 1
-
-    # Maximum number of channels to put in the available category
-    max_available: 2
-
-    # Maximum number of channels across all 3 categories
-    # Note Discord has a hard limit of 50 channels per category, so this shouldn't be > 50
-    max_total_channels: 20
-
-    # Prefix for help channel names
-    name_prefix: 'help-'
-
-    # Notify if more available channels are needed but there are no more dormant ones
-    notify: true
-
-    # Channel in which to send notifications
-    notify_channel: *DEV_LOUNGE_HELPERS_ORG
-
-    # Minimum interval between helper notifications
-    notify_minutes: 5
-
-    # Mention these roles in notifications
-    notify_roles:
-        - *HELPERS_ROLE
-
-python_news:
-    channel: *DEV_PY_NEWS
-    webhook: *PYNEWS_WEBHOOK
-
-##### <<  Add any additional sections you need to override from config-default.yml  >> #####
-            </code>
-          </pre>
-</div></div></div>
-<br>
+**Notes**:
+* Both `.env` and `.env.server` are and should remain ignored by git, otherwise you risk pushing sensitive information.
+* Skip the following step if you would like to configure the bot manually, but that will require more work.
```

Comment:
These don't appear as bulleted points, I think you need another newline before them

Reply:
Strange, it rendered just fine for me on Pycharm, that's why
![image](https://user-images.githubusercontent.com/48383734/224564188-8c2d4f5d-4ec1-4b73-9bb9-339ddeec5783.png)

Reply:
fixed in 2ce0b6936e93271e1de7f772686fb5adff7bb770